codegen_glibmm/templates/stub.h.templ

```diff
@@ -32,7 +32,7 @@ protected:
         {% for arg in method.in_args %}
         {{ arg.cpptype_in }} {{ arg.name }},
         {% endfor %}
-        MethodInvocation invocation) = 0;
+        const MethodInvocation &invocation) = 0;
```

Comment:
this will be altered like 
`invocation.ret(something)` right. I wonder if it works this way.

Reply:
Yes, true. It does not alter MethodInvocation itself though but the object contained in the RefPtr. But it is a bit confusing perhaps, I agree. Went back and forth on it but const:ified it so I would not have to create a temp variable.

Removed const from MethodInvocation methods and changed this to use a temporary instead so that we can use a & and not a const &.

Reply:
> I wonder if it works this way.

But yes, it worked. :)